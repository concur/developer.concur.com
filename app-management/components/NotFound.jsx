import React from 'react';

const NotFound = () => (
  <div className="wrapper">
    <div className="height-100" />
    <div className="text-center">
      <h1>404</h1>
      <h4>Sorry, that page doesn&#39;t exist.</h4>
      <a href="/index.html">Back to home</a>
    </div>
    <div className="height-100" />
  </div>
);

export default NotFound;
