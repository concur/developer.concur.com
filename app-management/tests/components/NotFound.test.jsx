import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import NotFound from '../../components/NotFound';

describe('<NotFound />', () => {
  it('renders with 404 text', () => {
    const page = shallow(<NotFound />);

    expect(shallowToJson(page)).toMatchSnapshot();
  });
});
