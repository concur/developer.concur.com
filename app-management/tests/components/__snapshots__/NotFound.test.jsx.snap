exports[`<NotFound /> renders with 404 text 1`] = `
<div
  className="wrapper">
  <div
    className="height-100" />
  <div
    className="text-center">
    <h1>
      404
    </h1>
    <h4>
      Sorry, that page doesn\'t exist.
    </h4>
    <a
      href="/index.html">
      Back to home
    </a>
  </div>
  <div
    className="height-100" />
</div>
`;
